aimmo-game/tests/functional/test_movements_in_map.py

```diff
@@ -14,15 +16,19 @@ class TestMovementsInMap(TestCase):
         'OBSTACLE_RATIO': 0,
     }
 
-    def set_up_environment(self, dummy_list=None, location=Location(0, 0), map_generator_class=map_generator.Main):
+    def set_up_environment(self, dummy_list=None, location=Location(0, 0),
```

Comment:
maybe we can make it consistent and have a new line per parameter?

Reply:
I considered this but it seems like a bit of overkill, I only go to the next line when I run out of 90 character limit. 

cc @CelineBoudier watchu say

Reply:
i see your point @mrniket but it might indeed be an overkill. If there are a lot of parameters, it doesn't improve the readability IMO

Reply:
fair enough, for me it improves readability if there are more than a certain number of parameters but if it's just me then I guess it's okay 👍 . Also if there are a lot of parameters I would argue splitting the function up or grouping the parameters together